src/bun.js/bindings/bindings.cpp

```diff
@@ -1685,6 +1805,19 @@ void JSC__JSFunction__optimizeSoon(JSC__JSValue JSValue0)
     JSC::optimizeNextInvocation(value);
 }
 
+bool JSC__JSFunction__getSourceCode(JSC__JSValue JSValue0, ZigString* outSourceCode)
```

Comment:
Do we need to do this? `toString` on a function returns the source code for the function